### DISCUSSION TITLE
Handle IPv6 snake URLs

### DISCUSSION THREAD

**Discussion by noveb** [BattlesnakeOfficial/feedback#47] [Feature Requests]
Good Morning from Germany :) 

I tried to add my snake with a IPv6 URL like http://[2a0a:a540:xxxx:xxxx:xxxx:f208:977b:c42c]:5000 but it did not work. The input field validates the URL as correct, but the snake is then unreachable from the BS servers.

It would be nice if this is supported in the future.

**Comment by chris-bsnake** [comment]
This is a known issue in the game engine that we are hoping to resolve in the next couple of months.

**Comment by bvanvugt** [comment]
@noveb thanks for bringing this up again! We're going to get this fixed in the next few... weeks. 

Creating Issue to track going forward.

**Reply by bvanvugt** [reply]
Tracked in issue #70

**Comment by noveb** [comment]
Hello @bvanvugt and @chris-bsnake , is there a way to check if issue #70 is resolved?

**Reply by bvanvugt** [reply]
@noveb Ack, yeah. #70 was deleted when we moved issue tracking -- sorry for the misdirection. This hasn't been fixed yet, but is scheduled for later this month / early Oct. We'll update this ticket once it's live!

**Comment by robbles** [comment]
@noveb We now have support for IPv6 via the new beta engine regions. The default AWS US-WEST-2 region does not yet support IPv6.

https://docs.battlesnake.com/references/engine-regions

The "GCP EUROPE-WEST4" region should work with your Battlesnake in Germany 😄 
Let us know if you run into any problems with it!
